newIDE/app/src/MainFrame/index.js

```diff
@@ -2022,6 +2026,29 @@ const MainFrame = (props: Props) => {
     openExternalLayout(name);
   };
 
+  const onEventsBasedObjectChildrenEdited = () => {
+    const { editorTabs } = state;
+    for (const editor of editorTabs.editors) {
+      const { editorRef } = editor;
+      if (editorRef) {
+        editorRef.onEventsBasedObjectChildrenEdited();
+      }
+    }
+  };
+
+  const onSceneObjectEdited = (
+    scene: gdLayout,
+    objectWithContext: ObjectWithContext
+  ) => {
+    const { editorTabs } = state;
+    for (const editor of editorTabs.editors) {
+      const { editorRef } = editor;
+      if (editorRef) {
+        editorRef.onSceneObjectEdited(scene, objectWithContext);
+      }
+    }
+  };
```

Comment:
Let's React.useCallback these functions to make them as "stable" as we can.

Reply:
Why do some functions not use `React.useCallback` either? Do their dependencies change at every rendering or something like this?